lang/src/main/java/org/kaazing/k3po/lang/internal/parser/ScriptParseStrategy.java

```diff
@@ -866,6 +872,12 @@ public AstConnectNode visitConnectNode(ConnectNodeContext ctx) {
             if (barrier != null) {
                 node.setBarrier(barrier.getText());
             }
+            RobotParser.LocationContext transport = ctx.value;
```

Comment:
Update import to let `LocationContext` be used directly here, same as other `RobotParser.*` types.